ts/protocol/otf_message_handler.py

```diff
@@ -310,10 +312,21 @@ def _retrieve_input_data(conn):
 
     length = _retrieve_int(conn)
     value = _retrieve_buffer(conn, length)
-    if content_type == "application/json" and (decode_req is None or decode_req == "true"):
-        model_input["value"] = json.loads(value.decode("utf-8"))
-    elif content_type.startswith("text") and (decode_req is None or decode_req == "true"):
-        model_input["value"] = value.decode("utf-8")
+    if content_type == "application/json" and (
+        decode_req is None or decode_req == "true"
+    ):
+        try:
+            model_input["value"] = json.loads(value.decode("utf-8"))
+        except Exception as e:
+            logging.info(f"Exception {e} thrown on value: {value}")
+    elif content_type.startswith("text") and (
+        decode_req is None or decode_req == "true"
+    ):
+        try:
+            model_input["value"] = value.decode("utf-8")
+        except Exception as e:
+            logging.info(f"Exception {e} thrown on value: {value}")
```

Comment:
If there is an error, what should value be populated as?

Reply:
Made this a bit clearer now, I'd like to return nothing if there's any issue